gamla/io_utils.py

```diff
@@ -233,3 +233,21 @@ async def post_json_with_extra_headers_async(
 #:
 #:    >>> response = post_json_async(30, "https://www.someurl.com/post_data", { "name": "Danny" })
 post_json_async = post_json_with_extra_headers_async({})
+
+
+@currying.curry
+async def post_params_with_extra_headers_async(
```

Comment:
can we use the implementation of `post_json_with_extra_headers_async` ?

i.e. change `post_json_with_extra_headers_async` to `post_json_with_extra_headers_and_params_async`

and then `post_json_with_extra_headers_async = post_json_with_extra_headers_and_params_async({})`

where the first argument should be `params`

Reply:
That's actually how I did it originally... wouldn't it be more cumbersome? because then one argument will always have to be `None`. Also, this will also require changing all the calls to `post_json_with_extra_headers_async` to add this parameter